### PR TITLE
Added Run-Loop

### PR DESCRIPTION
**Pull request by devn0ll**
Short Description: 

modified run script wich let you run the boot in a loop(if it crashes it restarts)

### CODE REVIEW

File: run_loop.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        echo -e "Please create one or use another config file"
+        echo -e "./run.sh [path/to/config/file]"
+        exit 1
+    fi
+fi
+while [ true  ]
+do
+python pokecli.py --config ${config}
+echo "##################################"
+echo "##Exit two times to end the loop##"
```
Comment (solderzzc) [inline run_loop.sh]: Change 'Exit' to  [Ctl+C] would be better?

Comment (devn0ll) [inline run_loop.sh]: Yeah sounds good

Comment (douglascamata) [inline run_loop.sh]: why don't you edit run.sh with these contents?

Comment (devn0ll) [inline run_loop.sh]: unsure if everyone wants to run theri bot forever ?

**Comment by solderzzc** [comment]
Thanks

**Comment by ZeroWolfe** [comment]
the code keep launching pokecli.py every 10 second

**Comment by devn0ll** [comment]
bashe is used to wait until the task is done and will continue then as long as you don't use &
so basicly it will do the follow
start HERE
print the message (when this is done)
wait one second (when this is done)
launch the bot (when this is done [means if it crashes or the python script is stopped ])
wait 10 seconds (when this is done)
go to HERE

**Pull request merged**